calculator.feature

```diff
@@ -0,0 +1,33 @@
+Feature: windows calculator
+
+Scenario: calc automation
+* configure robot = { highlight: true }
+* robot { fork: 'C:/Windows/System32/calc.exe' }
+
+#Addition operation
+* click('Clear')&&click('One')&&click('Plus')&&click('Two')&&click('Equals')
```

Comment:
This is fine. But can you optimize this code?. Basically use looping here to do all the operations.